src/test/java/com/artipie/asto/S3StorageTest.java

```diff
@@ -91,6 +91,25 @@ void shouldNotExistForUnknownObject(final AmazonS3 client) {
         MatcherAssert.assertThat(exists, Matchers.equalTo(false));
     }
 
+    @Test
+    void shouldMoveObject(final AmazonS3 client) throws Exception {
+        final String bucket = UUID.randomUUID().toString();
+        client.createBucket(bucket);
+        final byte[] original = "something".getBytes();
+        final String source = "source";
+        client.putObject(bucket, source, new ByteArrayInputStream(original), new ObjectMetadata());
+        final String destination = "destination";
+        new BlockingStorage(this.storage(bucket)).move(
+            new Key.From(source),
+            new Key.From(destination)
+        );
+        final byte[] downloaded;
+        try (S3Object s3Object = client.getObject(bucket, destination)) {
+            downloaded = ByteStreams.toByteArray(s3Object.getObjectContent());
+        }
+        MatcherAssert.assertThat(downloaded, Matchers.equalTo(original));
```

Comment:
@olegmoz See if you can inline `downloaded` object into the assertion; it is used only once. Also, please add a assertion message failure and avoid using static matchers.

Reply:
@paulodamaso inlined the variable and removed static matcher. Won't add reason message though, figured out with @g4s8 they make no sense when there is only one assetion. See https://github.com/artipie/artipie/pull/37

Reply:
@olegmoz I see. Well, just for the record, I really prefer to read a human-friendly sentence `"Method was not able to do what it was intended"` then decipher it from a method name `shouldDoWhatIsIntended`. An human readable sentence helps us to discover immediately why the test failed. I'd like to ask you and @g4s8 to reconsider about [artipie/artipie#37](https://github.com/artipie/artipie/pull/37), but I won't block this issue with this concern